### ISSUE TITLE
`sources`, `file` discarded after merge

### ISSUE THREAD

**Issue by jonathantneal** [keik/merge-source-map#3]
Hello! First, thank you so much. I love this module because it has easily allowed me to use PostCSS and Sass alongside each other without losing source maps as I move between them.

I am having an issue with losing `sources` and `file` after a merge.

I start with this source map:

```js
// map1
{
  version: 3,
  file: 'imports.result.css',
  sources: [ 'imports.css' ],
  sourcesContent: [ '@import "components/vars";\n@import "components/page";\n' ],
  names: [],
  mappings: 'AAAA,0BAA0B;AAC1B,0BAA0B'
}
```

Then, after running the CSS through node-sass, those `@import` statements are transpiled, and I have this source map:

```js
// map2
{
  version: 3,
  file: 'imports.result.css',
  sources: [ 'imports.css', 'components/vars.css', 'components/page.css' ],
  sourcesContent: 
   [ '@import "components/vars";\n@import "components/page";\n',
     '$font-stack:    Helvetica, sans-serif;\n$primary-color: #333;\n',
     ':root {\n\tcolor: $primary-color;\n\tfont: 100% $font-stack;\n}\n' ],
  names: [],
  mappings: 'AEAA,AAAA,KAAK,CAAC;EACL,KAAK,EDAU,IAAI;ECCnB,IAAI,EAAE,IAAI,CDFK,SAAS,EAAE,UAAU,GCGpC'
}
```

I use merge-source-map to merge these 2 source maps:

```
import mergeSourceMap from 'merge-source-map';

mergeSourceMap(map1, map2);
```

And the resulting source map has lost the `file` property as well as the `sources`: 

```js
{
  version: 3,
  sources: [ 'imports.css' ],
  sourcesContent: [ '@import "components/vars";\n@import "components/page";\n' ],
  sourceRoot: undefined,
  names: [],
  mappings: 'AAAA,KAAA,CAAA;EACA,KAAA,EAAA,IAAA;aADA,SAAA,EAA0B,UAAA'
}
```

What might I be doing wrong? Is merging a source map with possible sources possible with this module? Thanks for reviewing my issue! 😄 

---

Disclaimer: *I sorted the JSON properties in this issue to highlight which properties went missing*

**Comment by keik** [comment]
Thanks for reporting. Surely I could not consider it, maybe... I'll review on the weekend.

**Comment by keik** [comment]
@jonathantneal Can you try to use https://github.com/keik/merge-source-map/pull/4 and confirm improving?

**Comment by keik** [comment]
@jonathantneal I'm ready to merge and release it.

**Comment by jonathantneal** [comment]
Yes, this is very good.

**Comment by keik** [comment]
Thanks. Released as v1.1.0
